projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-replenishment-order-history.ts

```diff
@@ -102,7 +102,7 @@ export function verifySorting() {
 export function cancelReplenishmentInHistory() {
   const replenishmentHistoryAlias = visitReplenishmentHistory();
 
-  cy.get(replenishmentHistoryAlias).should((xhr) => {
+  cy.get(replenishmentHistoryAlias).then((xhr) => {
```

Comment:
Why change should for then?

Reply:
Restriction on 'should' on Cy12, No cy commands allowed inside a 'should', see: https://docs.cypress.io/guides/references/migration-guide#should
It needs to be replaced by 'then'.
I belive 'should' was use to have a 'replay' cabability:'special logic to rerun the callback function until no assertions throw within it.'
https://docs.cypress.io/api/commands/should#Whats-the-difference-between-then-and-shouldand